<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanListDialog</class>
 <widget class="QDialog" name="ScanListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Scan List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="scanListName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="priorityChannel50Label">
       <property name="text">
        <string>Priority Channel (50%)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="priorityChannel1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="priorityChannel25Label">
       <property name="text">
        <string>Priority Channel (25%)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="priorityChannel2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="transmitChannelLabel">
       <property name="text">
        <string>Transmit Channel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="transmitChannel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="channelList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="channelUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shared/resources.qrc">
           <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="channelDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shared/resources.qrc">
           <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+Down</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addChannel">
       <property name="text">
        <string>Add Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remChannel">
       <property name="text">
        <string>Remove Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt+-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../shared/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScanListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScanListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
